src/match-iterator.coffee

```diff
@@ -166,7 +182,11 @@ class ForwardsMultiLine
       match
 
 class BackwardsMultiLine
-  constructor: (@buffer, @regex, range, @chunkSize) ->
+  constructor: (@buffer, @regex, range, @options={}, @chunkSize) ->
+    if _.isNumber(@options)
+      @chunkSize = @options
+      @options = {}
```

Comment:
Why is a conditional needed here? Don't we always pass the `chunkSize` argument? I'd think that we could make `options` the last argument.